internal/backend/local/backend.go

```diff
@@ -92,6 +92,7 @@ type Local struct {
 }
 
 var _ backend.Backend = (*Local)(nil)
+var _ backendrun.OperationsBackend = (*Local)(nil)
```

Comment:
To be fair, this was already checked in a test: https://github.com/hashicorp/terraform/blob/0d6c1d315d6c6a32a3e2eb17d5fccf85bfd2e5b9/internal/backend/local/backend_test.go#L20-L24

But this assertion will run at compile time